### PR TITLE
Improve alagoasdevday website with relevant links

### PR DESCRIPTION
**Pull request by juarezpaf**
### Changed

- Remove subscription links and section
- Add slide links for most speakers

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -431,11 +428,11 @@ <h2 class="section-title">Programação</h2>
                   08h40
                 </div>
 
-                <div class="cd-timeline-img schedule-speaker">
-                  <div class="schedule-speaker-img">
+                <a href="http://twitter.com/juarezpaf" class="cd-timeline-img schedule-speaker">
```
Comment (pragmaticivan) [inline app/index.html]: We don't want to put links here.

Comment (juarezpaf) [inline app/index.html]: @pragmaticivan why not? I think it's something expected to have links in speakers avatars.

Comment (pragmaticivan) [inline app/index.html]: Actually We don't need links in any of the schedule-speaker timeline. Because all the links are on the Speaker List.

Comment (juarezpaf) [inline app/index.html]: @pragmaticivan @maclevison @gmmcal how are we going to add slide links into #add2015 website?

Comment (juarezpaf) [inline app/index.html]: Just talked with Carlyson! Doing changes here :dancers:

**Comment by juarezpaf** [comment]
@pragmaticivan ready to go!

**Comment by gmmcal** [comment]
Fixed on 5241949ba121d1345eac3db3021427b658a91370

**Pull request closed without merging**